src/pages/TeamSettings/Members/Members-Table.vue

```diff
@@ -346,16 +349,24 @@ export default {
       v-if="selectedUser"
       v-model="dialogRemoveUser"
       type="error"
+      :title="
+        deleteSelfWarning
+          ? '*** You are removing yourself from this tenant. ***'
+          : `Are you sure you want to remove ${selectedUser.email} from your team?`
+      "
       :dialog-props="{ 'max-width': '600' }"
       :disabled="isRemovingUser"
       :loading="isRemovingUser"
-      :title="
-        `Are you sure you want to remove ${selectedUser.email} from your team?`
-      "
       @confirm="removeUser(selectedUser.membershipId)"
     >
-      <span class="font-weight-bold">{{ selectedUser.username }}</span>
-      will no longer be able to access your team account.
+      <div v-if="deleteSelfWarning" class="red--text"
+        >Are you sure you want to remove yourself? You will no longer be able to
+        access this team account.
+      </div>
```

Comment:
```suggestion
        >Are you sure you want to remove yourself from {{ tenant.name }}? You'll no longer be able to access your Projects or Flows associated with {{ tenant.name }}.
      </div>
```
I'm thinking we can make this a little more explicit.